crates/bevy_winit/src/web_resize.rs

```diff
@@ -31,7 +31,7 @@ fn canvas_parent_resize_event_handler(
 ) {
     for event in resize_events.receiver.try_iter() {
         if let Some(window) = winit_windows.get_window(event.window) {
-            window.set_inner_size(event.size);
+            let _ = window.request_inner_size(event.size);
```

Comment:
Non-blocking, it's a bug in winit. This is so odd, but the method itself has side-effects, but the returned value is marked as `#[must_use]`.

Reply:
The method doesn't always have side effects. As stated in the documentation this won't always cause a `Resized` event to fire, so the returned `Option` has to be evaluated to figure out the new size or if the size has changed at all.